subsys/usb/class/cdc_acm.c

```diff
@@ -373,6 +373,21 @@ static void cdc_acm_int_in(u8_t ep, enum usb_dc_ep_cb_status_code ep_status)
 	LOG_DBG("CDC_IntIN EP[%x]\r", ep);
 }
 
+static void cdc_acm_reset_port(struct cdc_acm_dev_data_t *dev_data)
+{
+	k_sem_give(&poll_wait_sem);
+	dev_data->rx_ready = 0;
+	dev_data->tx_ready = 0;
+	dev_data->tx_irq_ena = 0;
+	dev_data->rx_irq_ena = 0;
+	dev_data->line_coding.dwDTERate = sys_cpu_to_le32(115200);
+	dev_data->line_coding.bCharFormat = 0;
+	dev_data->line_coding.bParityType = 0;
+	dev_data->line_coding.bDataBits = 8;
+	dev_data->serial_state = 0;
+	memset(&dev_data->rx_buf, 0, CDC_ACM_BUFFER_SIZE);
+}
```

Comment:
I think clearing LINE_STATE should be cleared too. Currently, once DTR is set, it's never released. See https://github.com/zephyrproject-rtos/zephyr/issues/15759

Reply:
Added